### PR TITLE
[IMP] point_of_sale: Product configurator

### PR DESCRIPTION
**Pull request by aprieels**
Configure products with create_variants = 'never' from the POS

TaskID: 208205

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/views/pos_config_view.xml [hunk 1]
```diff
@@ -96,6 +96,17 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="col-12 col-lg-6 o_setting_box">
```
Comment (pimodoo) [inline addons/point_of_sale/views/pos_config_view.xml:99]: Est ce qu'on ne devrait pas montrer cette option que quand les variants sont activés dans les settings?

**Comment by aprieels** [comment]
@tde-banana-odoo I've moved some fields from the `sale` module to `product` so that they could be used in `point_of_sale` too. Could you have a look and tell me if this works for you?

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Comment by pimodoo** [comment]
@robodoo r-

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Comment by vedler** [comment]
This PR reintroduces the bug fixed in #41913 (fiscal position taxes computed twice) on [Line 2897 in POS models.js](https://github.com/odoo-dev/odoo/blob/2f914bf81357136dd85f0c67aa321d8074048b36/addons/point_of_sale/static/src/js/models.js#L2897), I'll try making a fix with a new PR soon unless someone from Odoo team wants to go for it themselves

Tried with variants enabled, fiscal position price included mapped to price excluded, which works properly in Sale, but computes tax twice in PoS.

@agr-odoo  @nle-odoo  @aprieels

**Comment by nle-odoo** [comment]
Hi @vedler 


I guess the issue seems to be that in:

```js
line.set_unit_price(line.get_unit_price() + options.price_extra);
this.fix_tax_included_price(line);
```

we already applied `fix_tax_included_price` to the price gotten by `line.get_unit_price()` so we are applying position two times on original price, and one time to price_extra.

**Pull request closed without merging**